book/chapters/chapter11/large-scale_benchmarking.qmd

```diff
@@ -110,7 +110,7 @@ odatasets = list_oml_data(
   number_features = c(10, 20), # 10-20 features
   number_instances = c(45000, 50000), # 40,000-50,000 rows
   number_classes = 2 # 2 classes
-)[, c("data_id", "name")]
+)
```

Comment:
Does removing this cause too much printout?

Reply:
this did not produce output (due to a bug in mlr3oml which returns the table invisible).

Instead, the subsetted table is now explicitly printed a few lines later.

Reply:
Okay thanks